### ISSUE TITLE
ToDos

### ISSUE THREAD

**Issue by stark-t** [stark-t/PAI_diptera#8]
- [x] mean training time
- [x] epoch-step number of min validation score
- [x] image augmentations
- [x] uncertainty estimates
  - [x] augmentations at test time
  - [x] monte carlo dropout
- [x] pre-trained models
- [x] use any/many timm models
- [ ] compare best fitnes vs total runtime
- [ ] visualize batch augmentations
